internal/deviations/deviations.go

```diff
@@ -1462,3 +1462,8 @@ func UseOldOCPathStaticLspNh(dut *ondatra.DUTDevice) bool {
 func ConfigLeafCreateRequired(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetConfigLeafCreateRequired()
 }
+
+// StaticMplsLspOCUnsupported returns true if static mpls lsp parameters are unsupported
+func StaticMplsLspOCUnsupported(dut *ondatra.DUTDevice) bool {
```

Comment:
as noted elsewhere, I think this is a duplicate of a deviation already defined at `StaticMplsOCUnsupported`